src/main/java/uk/gov/pay/card/model/CardInformationRequest.java

```diff
@@ -6,15 +6,14 @@
 import javax.validation.constraints.Size;
 
 import static java.lang.String.format;
-import static uk.gov.pay.card.db.CardInformationStore.CARD_RANGE_LENGTH;
 
 public class CardInformationRequest {
 
     /**
-     * PANs can be between 10 and 19 characters
+     * PANs can be between 8 and 19 characters
      * @see <a href="https://www.ansi.org/news_publications/news_story?articleid=da7bcb04-0654-4e03-af54-0e55d50b93a8">ANSI IIN</a>
      */
-    @JsonProperty("cardNumber") @NotEmpty @Size(min = CARD_RANGE_LENGTH, max = 19)
+    @JsonProperty("cardNumber") @NotEmpty @Size(min = 8, max = 19)
```

Comment:
think we should keep this at 11 for now, we can always change it after we've validated the new behaviour